hafas_fetcher.py

```diff
@@ -154,17 +162,18 @@ def write_json(self):
                 "icon": dep.category_icon,
                 "id": dep.id,
                 "next_time": (
-                    dep.follow.realtime.strftime("%H:%M") if dep.follow else ""
+                    dep.follow.realtime.astimezone(self.tz).strftime("%H:%M") if dep.follow else ""
```

Comment:
Please use `timezone.localize()` instead of `astimezone()`. `.astimezone()` is not aware of the date/time it actually contains, but instead just uses the first offset that's in the database, resulting in +00:53 for Europe/Berlin time.

Reply:
This shouldn't be an issue because tzinfo in `HAFASEvent` is set either based on tz info from HAFAS or `CONFIG["timezone"]` if not available from HAFAS. Honestly have no idea where +00:53 is coming from.

Reply:
Ah, turns out the problem was in `HAFASEvent` and not `HAFASFetcher`. Should be fixed now.

Reply:
We ran into a very similar problem already (if not exactly the same one), see https://chaos.social/@rami/111651754744821928 for more context.

Reply:
/me shakes fist at timezones